spec/pluralrules.html

```diff
@@ -0,0 +1,360 @@
+      <emu-table id="table-pluralrules-resolvedoptions-properties">
+        <emu-caption>Resolved Options of PluralRules Instances</emu-caption>
+        <table class="real-table">
+          <thead>
+            <tr>
+              <th>Internal Slot</th>
+              <th>Property</th>
+            </tr>
+          </thead>
+          <tr>
+            <td>[[Locale]]</td>
+            <td>"locale"</td>
+          </tr>
+          <tr>
+            <td>[[Type]]</td>
+            <td>"type"</td>
+          </tr>
+          <tr>
+            <td>[[MinimumIntegerDigits]]</td>
+            <td>"minimumIntegerDigits"</td>
+          </tr>
+          <tr>
+            <td>[[MinimumFractionDigits]]</td>
+            <td>"minimumFractionDigits"</td>
+          </tr>
+          <tr>
+            <td>[[MaximumFractionDigits]]</td>
+            <td>"maximumFractionDigits"</td>
+          </tr>
+          <tr>
+            <td>[[MinimumSignificantDigits]]</td>
+            <td>"minimumSignificantDigits"</td>
+          </tr>
+          <tr>
+            <td>[[MaximumSignificantDigits]]</td>
+            <td>"maximumSignificantDigits"</td>
+          </tr>
+          <tr>
+            <td>[[PluralCategories]]</td>
+            <td>"pluralCategories"</td>
+          </tr>
+        </table>
+      </emu-table>
+    </emu-clause>
+  </emu-clause>
+
+  <emu-clause id="sec-properties-of-intl-pluralrules-instances">
+    <h1>Properties of Intl.PluralRules Instances</h1>
+
+    <p>
+      Intl.PluralRules instances inherit properties from %PluralRulesPrototype%.
+    </p>
+
+    <p>
+      Intl.PluralRules instances have an [[InitializedPluralRules]] internal slots.
+    </p>
+
+    <p>
+      Intl.PluralRules instances also have several internal slots that are computed by the constructor:
+    </p>
+
+    <ul>
+      <li>[[Locale]] is a String value with the language tag of the locale whose localization is used by the plural rules.</li>
+      <li>[[Type]] is one of the String values `"cardinal"`, or `"ordinal"`, identifying the plural rules used.</li>
+      <li>[[MinimumIntegerDigits]] is a non-negative integer Number value indicating the minimum integer digits to be used.</li>
+      <li>[[MinimumFractionDigits]] and [[MaximumFractionDigits]] are non-negative integer Number values indicating the minimum and maximum fraction digits to be used. Numbers will be rounded or padded with trailing zeroes if necessary.</li>
+      <li>[[MinimumSignificantDigits]] and [[MaximumSignificantDigits]] are positive integer Number values indicating the minimum and maximum fraction digits to be used. Either none or both of these properties are present; if they are, they override minimum and maximum integer and fraction digits.</li>
+      <li>[[PluralCategories]] is an array of unique string values, from the the list `"zero"`, `"one"`, `"two"`, `"few"`, `"many"` and `"other"`, that are relevant for the locale whose localization is specified in LDML Language Plural Rules.</li>
```

Comment:
- I'd remove "unique", because it makes it sound like that string values themselves can be unique or non-unique. Instead of that I'd simply say that the array must not contain any duplicates, because I think that's what was meant by "unique" here.
- "LDML Language Plural Rules" is never defined.
- The complete sentence reads a bit strange: "[The] [...] values [...] are relevant for the locale whose localization is specified in LDML Language Plural Rules." This should be reworded to make it more clear where and why these values are used.

Reply:
I will ask for clarifications on this points tomorrow.